src/runtimes/node/bundlers/esbuild/bundler_target.ts

```diff
@@ -10,18 +13,41 @@ const versionMap = {
 type VersionKeys = keyof typeof versionMap
 type VersionValues = typeof versionMap[VersionKeys]
 
-export const getBundlerTarget = (suppliedVersion?: string): VersionValues => {
+const getBundlerTarget = (suppliedVersion?: string): VersionValues => {
   const version = normalizeVersion(suppliedVersion)
 
   if (version && version in versionMap) {
     return versionMap[version as VersionKeys]
   }
 
-  return DEFAULT_VERSION
+  return versionMap[`${DEFAULT_NODE_VERSION}.x`]
+}
+
+const getModuleFormat = async (
+  srcDir: string,
+  featureFlags: FeatureFlags,
+  configVersion?: string,
+): Promise<{ includedFiles: string[]; moduleFormat: ModuleFormat }> => {
+  const packageJsonFile = await getClosestPackageJson(srcDir)
```

Comment:
I'm not sure why we need to use `getClosestPackageJson` in this case. `srcDir` would be smth like `projectDir/netlify/functions/my-function.ts`, right? The difference between `getClosesPackageJson` and `getPackageJson` is that one has an optimisation to not read further than the nearest `node_modules` folder, which wouldn't be the case here. What am I missing?

Reply:
`getClosestPackageJson` returns the contents of the `package.json` _and_ its path, whereas `getPackageJson` returns just the contents. I agree that we should consolidate these, but I'd prefer to do that as a separate PR to reduce the amount of changes. What do you think?

Reply:
oh, somehow missed this reply. agree! I'll work on that.